src/components/TagInput/TagInput.tsx

```diff
@@ -17,6 +18,7 @@ export const TagInput = forwardRef<HTMLInputElement, TagInputProps>(({
   hint,
   label,
   onTagsChange,
+  placeholder = 'Press ENTER to search for serveral terms',
   tags,
 }) => {
   const [input, setInput] = useState('');
```

Comment:
In handleKeyDown, do not use e.keyCode but e.key
I think for accessibility, Tab should not handle new key